prototypes/styletransfer/model.py

```diff
@@ -67,10 +68,10 @@ def feature_representations(
     ) -> Tuple[tf.Tensor, tf.Tensor]:
 
         content_rep, style_rep = self.feature_representations_batch(
-            np.expand_dims(content_img, 0), np.expand_dims(style_img, 0)
+           content_img, style_img
         )
 
-        return content_rep[0], style_rep[0]
+        return content_rep, style_rep
```

Comment:
Oh I understand now, `content_rep` e `style_rep` are lists, one for each layer we are interested in.